### PR TITLE
docs(CHANGES): reflect recent changes.

### PR DESCRIPTION
**Pull request by marcoscaceres**
Some recent pull requests that were merged were not reflected here.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -460,7 +468,7 @@ Notable fixes and improvements to existing languages:
 - Added `abstract` and `namespace` keywords to TypeScript by [Daniel Rosenwasser][]
 - Added `label` support to Dockerfile by [Ladislav Prskavec][]
 - Crystal highlighting improved by [Tsuyusato Kitsune][]
-- Missing Swift keywords added by [Nate Cook][]
+- Missing Swift keywords added by []
```
Comment (IIIIllllIIIIllllIIIIllllIIIIllllIIIIll) [inline CHANGES.md]: Do we leave this unattributed?

Comment (marcoscaceres) [reply CHANGES.md]: we did

Comment (marcoscaceres) [reply CHANGES.md]: accident... fixing.

**Comment by marcoscaceres** [comment]
@IIIIllllIIIIllllIIIIllllIIIIllllIIIIll should be good now 🤞

**Comment by marcoscaceres** [comment]
> Am not reviewing for completeness, but the additions LGTM

That's fine... it's a best effort thing.

**Pull request merged**